README.md

```diff
@@ -200,6 +200,11 @@ When outputting in a human-readable format, legitify support the conventional `-
 ### Misc
 
 - Use the `--failed-only` flag to filter-out passed/skipped checks from the result.
+- Use the `--ignore-policies POLICY_NAME1, PN2` to ignore specific policies, e.g. 
```

Comment:
I think we should allow reading from a file the ignored policies.
Otherwise, we will have a really cluttered CLI